docs/product/adrianneb/variants-revamp-v3/variant_option_management.md

```
@@ -0,0 +1,276 @@
+---
+id: variant-option-management
+title: Variant Option Management
+sidebar_label: Variant Options
+sidebar_position: 4
+---
+
+# Variant Option Management
+
+#### **UC 14 | Add a New Variant Option**
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)